alerts/elasticsearch/elasticsearch-red-cluster-status.json

```diff
@@ -0,0 +1,34 @@
+{
+  "displayName": "Elasticsearch - red cluster status",
+  "documentation": {
+    "content": "When red cluster status duration is exceeds minute, at least one primary shard is missing, and you are missing data.",
```

Comment:
When we update the text above, we can update the text here as well

Reply:
fixed!